.github/workflows/release-asset-publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish release .jar
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Publish release .jar
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 17 ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          cache: 'maven'
+          java-version: ${{ matrix.java }}
```

Comment:
Do you really need to release different jars for all java versions?

Reply:
No, at least for now. But all our actions uses matrix-like java version definition, so I decided to use it here as well.